src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -116,13 +116,13 @@
               </template>
 
               <!-- Firm Name Change confirmation -->
-              <template v-if="(isProprietor || isPartner) && !isNaN(activeIndex) && !isConversionFiling">
+              <template v-if="isChangeFiling && (isProprietor || isPartner) && !isNaN(activeIndex)">
```

Comment:
It's safer to target a specific filing type instead of "not some other filing type" (for future dev).